### PR TITLE
Chore/clean test suite

### PR DESCRIPTION
**Pull request by bokay**
This PR will fix the current test suite, so we can proceed with adding code safely.

### CODE REVIEW

**Comment by bokay** [comment]
@poulping or @DanielWright do you mind reviewing this PR please ?

File: lib/ship_quiet_logistics/documents/shipment_order.rb [hunk 1]
```diff
@@ -94,6 +94,14 @@ def date_stamp
       def order_type
         'SO'
       end
+
+      def country_name(address)
```
Comment (DanielWright) [inline lib/ship_quiet_logistics/documents/shipment_order.rb]: Does this belong in this PR? It seems out of scope.

File: lib/ship_quiet_logistics/documents/shipment_order.rb [hunk 2]
```diff
@@ -94,6 +94,14 @@ def date_stamp
...
+        if address.respond_to?(:country_name)
+         address.country_name
+        elsif address.country.present?
+          address.country.name
+        end
```
Comment (DanielWright) [inline lib/ship_quiet_logistics/documents/shipment_order.rb]: Is `nil` an okay result, if `address` neither responds to `country_name` or `country`?

File: spec/features/sends_rma_document_spec.rb [hunk 3]
```diff
@@ -1,7 +1,12 @@
 require 'spec_helper'
 
 RSpec.describe 'Sending RMA Documents', :vcr do
-  let(:rma) { decorate(create(:return_authorization))  }
+  let(:rma) { decorate(create(:return_authorization, number: 'RA368041525'))  }
```
Comment (DanielWright) [inline spec/features/sends_rma_document_spec.rb]: Would these values make more sense in the `RMADecoratorStub`, since they aren't specific to this spec? I tend to prefer that strings or numbers in specs have expectations set against them, rather than just being part of the test setup. Meanwhile, the `RMADecoratorStub` exists precisely so the `rma` returns the "correct" values.

File: lib/ship_quiet_logistics/documents/shipment_order.rb (hunk 1 above)
Comment (bokay) [reply lib/ship_quiet_logistics/documents/shipment_order.rb]: That's right, it shouldn't.
Originally, it was using `country_name` which is a decorator setup in Aday's project. We should use the bare behaviour of Spree instead: `country.name` as this gem is tested upon a dummy Spree project.

File: lib/ship_quiet_logistics/documents/shipment_order.rb (hunk 2 above)
Comment (bokay) [reply lib/ship_quiet_logistics/documents/shipment_order.rb]: I don't know! Maybe ?
Would you raise an error for this ?

**Comment by bokay** [comment]
@DanielWright The two last commits should address the issues you mentioned.

**Comment by bokay** [comment]
Still relevant. I just never got feedback.
